code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -1,31 +1,33 @@
+//Catagories; Xenowear - Sinta / Xenowear - Tajara  / Xenowear - Skrell / Xenowear - Vaurca / Xenowear - IPC / Xeno wear
+
 //unathi items
 
 /datum/gear/suit/unathi_mantle
 	display_name = "hide mantle (Unathi)"
 	path = /obj/item/clothing/suit/unathi/mantle
 	cost = 1
 	whitelisted = list("Unathi")
-	sort_category = "Xenowear"
+	sort_category = "Xenowear - Sinta"
```

Comment:
This should probably read the common name of the species (Xenowear - Unathi), as that follows the pattern established by the other categories (ie it's Xenowear - Vaurca not Xenowear - Vaur'uyit'zir)

Reply:
agree